recipe/conda_build_config.yaml

```diff
@@ -1,8 +1,15 @@
+YEAR:
+  - 2017
+VER:
+  - 15
+
 WINSDK_VERSION:
-  - 10.0.17134.0
+  - 10.0.17763.0
 
 MSVC_HEADERS_VERSION:
   - 14.11.25547
+# We would like to upgrade this but sadly don't find a compatible version currently
+#  - 14.16.27012
```

Comment:
We can upgrade this to 14.14 with https://visualcpp.myget.org/feed/dailymsvc/package/nuget/VisualCppTools.Community.Daily.VS2017Layout

Reply:
Looks like https://github.com/mstorsjo/msvc-wine/blob/master/vsdownload.py supports multiple versions.

Reply:
It downloads `14.16.27023` whereas on Azure VMs we only have `14.16.27012`. This will always diverge as `vsdownload.py` is only able to download the latest patch release.

This PR already updates to 14.14. Would it be ok to move the integration of `vsdownload` into a separate one and merge this?

Reply:
On azure VMs we have 14.16.27023 MSVC, but 14.16.27012 runtime version, but sure let's update to 14.14 and do 14.16 later